src/menus/menu.jsx

```diff
@@ -249,10 +245,10 @@ const Menu = React.createClass({
     let selectedChildrenStyles = {};
 
     if (selected) {
-      selectedChildrenStyles = this.mergeStyles(styles.selectedMenuItem, selectedMenuItemStyle);
+      selectedChildrenStyles = Object.assign(styles.selectedMenuItem, selectedMenuItemStyle);
     }
 
-    let mergedChildrenStyles = this.mergeStyles(
+    let mergedChildrenStyles = Object.assign(
       child.props.style || {},
```

Comment:
hmmmm this is dangerous. `Object.assign({}, child.props.style, selectedChildrenStyles)`